data/db/mil-tamron.xml

```diff
@@ -224,5 +224,56 @@
 			<distortion focal="20" model="ptlens" a="0.02545" b="-0.10238" c="0.04699" />
         </calibration>
     </lens>
+    
+    <lens>
+        <maker>Tamron</maker>
+        <model>E 17-28mm F2.8-2.8</model>
+        <mount>Sony E</mount>
+        <aperture max="2.8" min="2.8"/>
+        <cropfactor>1</cropfactor>
+        <calibration>
+        	<!-- Thanks to fheinig for the profile -->
+	        <distortion model="ptlens" focal="17" a="0.0119487" b="-0.0187828" c="-0.0379118"/>
+	        <distortion model="ptlens" focal="19" a="0.0158913" b="-0.030955" c="-0.0138245"/>
+	        <distortion model="ptlens" focal="20" a="0.0173522" b="-0.0355207" c="0.00462118"/>
+	        <distortion model="ptlens" focal="21" a="0.0163097" b="-0.0313141" c="0.00317845"/>
+	        <distortion model="ptlens" focal="22" a="0.014597" b="-0.026237" c="0.00275903"/>
+	        <distortion model="ptlens" focal="23" a="0.0147393" b="-0.0268074" c="0.00879658"/>
+	        <distortion model="ptlens" focal="24" a="0.0132051" b="-0.0221023" c="0.00665771"/>
+	        <distortion model="ptlens" focal="25" a="0.013138" b="-0.0229404" c="0.0112717"/>
+	        <distortion model="ptlens" focal="26" a="0.00474933" b="0.00263057" c="-0.0114455"/>
+	        <distortion model="ptlens" focal="28" a="0.0105916" b="-0.0179303" c="0.0119596"/>
+        </calibration>
+    </lens>
+    
+    <lens>
+        <maker>Tamron</maker>
+        <model>E 28-200mm F2.8-5.6 A071</model>
+        <mount>Sony E</mount>
+	      <aperture max="5.6" min="2.8"/>
```

Comment:
Some of these lines have got weird tab alignments. Would you mind sorting them out?